packages/mcp-docscribe/src/templates/prompt-templates.ts

```
@@ -0,0 +1,260 @@
+                return new UIUXPromptTemplate();
+            case 'audience':
+                return new AudiencePromptTemplate();
+            case 'accessibility':
+                return new AccessibilityPromptTemplate();
+            case 'api':
+                return new APIPromptTemplate();
+            default:
+                return new GenericPromptTemplate();
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)